_posts/2018-26-04-elixir-post.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title:  "Working With Object Oriented and Functional Programming Without Losing Your Mind"
+date:   2018-04-26 12:00:00 -0400
+categories: elixir coding ruby oo functional
+author: Tania Paiva
+---
+
+## Working With Object Oriented and Functional Programming Without Losing Your Mind
+
+At Forward Financing we work with several technologies, our stack is pretty mixed.
+We work with Rails, Phoenix, React, Redux among others, which means we are changing the
+development environment sometimes more than once at day.
+
+Sometimes in the morning you are developing a new feature in the Elixir-Phoenix environment, and in the
```

Comment:
Repetition of `sometimes`. Maybe something along the lines of
`In practice, this means you could find yourself developing a new feature in the Elixir-Phoenix environment in the morning, and improving existing Ruby on Rails code in the afternoon.`